src/handlers/check-modified-base-rate.ts

```diff
@@ -0,0 +1,38 @@
+import { Context } from "../types/context";
+import { isPushEvent } from "../types/typeguards";
+import { getCommitChanges } from "./get-commit-changes";
+
+export const ZERO_SHA = "0000000000000000000000000000000000000000";
+const BASE_RATE_FILES = [".github/ubiquibot-config.yml", ".github/.ubiquibot-config.yml"];
```

Comment:
We changed these to .ubiquity-os.config.yml. We also need to consider the dev environment now potentially which is .ubiquity-os.dev.config.yml I believe

Reply:
I wonder if these constants should be exported from the Kernel to avoid mistakes.

Reply:
It's a good idea

Reply:
https://github.com/ubiquity-os/ubiquity-os-kernel/pull/144